cli/templates/project/src/hooks/use-breakpoints.js

```diff
@@ -35,19 +35,24 @@ const BREAKPOINTS = [...BREAKPOINT_MAP].reduce((acc, curr) => {
  * {breakpointName === BREAKPOINTS.small && <p>This only renders at small</p>}
  */
 export function useBreakpoints() {
-  const [currentBP, setCurrentBP] = useState([null, null])
+  const [currentBP, setCurrentBP] = useState(BREAKPOINT_MAP.get("S"))
   const [breakpointIndex, breakpointName] = currentBP
 
-  const handleResize = useCallback((e) => {
+  const handleResize = useCallback((_) => {
     if (typeof window === "undefined") return
 
-    setCurrentBP(
-      BREAKPOINT_MAP.get(
-        window
-          .getComputedStyle(document.querySelector("body"), "::after")
-          .content.replace(/'|"/gi, "")
-      )
-    )
+    const body = document.querySelector("body")
+    if (!body) return
+
+    const styles = window.getComputedStyle(body, "::after")
+
+    if (!styles?.content) return
+
+    let breakpointSnipeContent = styles.content.replace(/'|"/gi, "")
+
+    if (![...BREAKPOINT_MAP.keys()].includes(breakpointSnipeContent)) breakpointSnipeContent = "S"
```

Comment:
Thanks @andrewrubin !
Could we simply try a `.get` on the map and if nothing returns we do an early return instead of doing the spread plus the lookup?

Reply:
Done!